lambda_functions/analyzer/main.py

```diff
@@ -155,7 +155,13 @@ def analyze_lambda_handler(event: Dict[str, Any], lambda_context: Any) -> Dict[s
             binary.save_matches_and_alert(
                 lambda_version, os.environ['YARA_MATCHES_DYNAMO_TABLE_NAME'],
                 os.environ['YARA_ALERTS_SNS_TOPIC_ARN'])
-
+        else:
+            LOGGER.info('%s did not match any YARA rules', binary)
+            binary.safe_alert_only(
+                lambda_version, 
+                os.environ['SAFE_SNS_TOPIC_ARN'])
```

Comment:
What happens if someone has _not_ enabled the safe topic? Either Terraform will fail or this ARN will be blank and the Lambda function will fail. You'll need to test and fix that case:

* Deploy with the `enable_safe_alerts` option set to 0
* `./manage.py live_test`